db/usertable/usertable.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS usertable;
+CREATE TABLE usertable (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) UNIQUE,
+    email VARCHAR(255) UNIQUE,
+    password VARCHAR(255),
+    code VARCHAR(50),
+    status VARCHAR(255),
+    user_type INT(1)
+);
+-- APART --- 
+-- Agregar la columna created_at a la tabla usertable
+ALTER TABLE usertable ADD created_at DATETIME DEFAULT CURRENT_TIMESTAMP;
```

Comment:
- tables declaration goes to the db/db.sql.
- users table already exists, so modified that instead of have multiple tables for the same purpose